File: frontends/llvm/tests/globals.c
int GLOBAL = 0;

// AARD: function: main
int main()
{
    // AARD: #1:1 -> #1:2  ::  defs: %1 / uses: %1 [@1 7:11-7:11]
    GLOBAL++;

    // AARD: #1:2 ->   ::  defs:  / uses:  [@1 10:5-10:5]  { ret }
    return 0;
}

// AARD: @1 = globals.c

File: frontends/python/tests/analysis/globals.py
# AARD: function: __main__

# AARD: #1:1 ->   ::  defs: %1 / uses:  [@1 4:1-4:8]
foo = 3

# AARD: function: get_foo
def get_foo():
    # AARD: #1:2 ->   ::  defs:  / uses: %1 [@1 9:5-9:15]  { ret }
    return foo

# AARD: @1 = globals.py
